src/data-access/FileManager/RouteUpload.js

```diff
@@ -0,0 +1,26 @@
+import * as auth from "solid-auth-client";
+import SolidFileClient from "solid-file-client";
+
+const fileClient = new SolidFileClient(auth, { enableLogging: true });
+
+
+export default{
+    main(name, text){
+        uploadRoute(name, text);
+    }
+}
+async function uploadRoute(name, text){
+  let session = await auth.currentSession();
+  if (!session || session.webId === undefined || session.webId === null) {
+    throw new Error("You are not logged in.");
+  }
+  var file = new File([text], name+".json", {
```

Comment:
In this line, the extension of the file should be `.jsonld` because later the type is stated as `application/ld+json`.